qa/suites/orch/cephadm/smoke/distro/centos_8.stream_container_tools_crun.yaml

```diff
@@ -0,0 +1 @@
+.qa/distros/podman/centos_8.stream_container_tools_crun.yaml
```

Comment:
Ok this is becoming super painful to backport, as we're mixing distro changes and suite changes here:

```
Traceback (most recent call last):
  File "/home/swagner/teuthology/virtualenv/bin/teuthology-suite", line 33, in <module>
    sys.exit(load_entry_point('teuthology', 'console_scripts', 'teuthology-suite')())
  File "/home/swagner/teuthology/scripts/suite.py", line 193, in main
    return teuthology.suite.main(args)
  File "/home/swagner/teuthology/teuthology/suite/__init__.py", line 144, in main
    run.prepare_and_schedule()
  File "/home/swagner/teuthology/teuthology/suite/run.py", line 398, in prepare_and_schedule
    num_jobs = self.schedule_suite()
  File "/home/swagner/teuthology/teuthology/suite/run.py", line 560, in schedule_suite
    seed=self.args.seed)
  File "/home/swagner/teuthology/teuthology/suite/build_matrix.py", line 57, in build_matrix
    mat, first, matlimit = _get_matrix(path, subset)
  File "/home/swagner/teuthology/teuthology/suite/build_matrix.py", line 67, in _get_matrix
    mat = _build_matrix(path, mincyclicity=outof)
  File "/home/swagner/teuthology/teuthology/suite/build_matrix.py", line 146, in _build_matrix
    fn)
  File "/home/swagner/teuthology/teuthology/suite/build_matrix.py", line 130, in _build_matrix
    item=fn)
  File "/home/swagner/teuthology/teuthology/suite/build_matrix.py", line 155, in _build_matrix
    return matrix.Sum(item, submats)
  File "/home/swagner/teuthology/teuthology/suite/matrix.py", line 251, in __init__
    f"Sum requires non-empty _submats: {item}"
AssertionError: Sum requires non-empty _submats: 0-distro
```
Would you mind creating two symlink targets for me? 

* `qa/distros/container-hosts` containing podman and docker distros
* `qa/distros/single-container-host`.

Reply:
what goes in ``single-container-host``?

Reply:
My idea is to avoid things like https://github.com/ceph/ceph/tree/240241dae589f1c6996a9011cbb42dc1792c3927/qa/suites/orch/cephadm/dashboard/0-distro

Reply:
See https://github.com/ceph/teuthology/pull/1696